### ISSUE TITLE
Athena data source schema viewer is missing tables

### ISSUE THREAD

**Issue by fbertsch** [mozilla/redash#371]
I thought this was a known issue but do not see an issue filed.

Basically a bunch of tables are missing from the Athena data source. This has the unfortunate side effect of pushing users to use the Presto data source instead, because they falsely believe their tables do not reside in Athena.

https://screenshots.firefox.com/7o0dkwcLhtynPpV3/sql.telemetry.mozilla.org

**Comment by jezdez** [comment]
So I think this is https://github.com/getredash/redash/issues/2500 basically which has a PR open https://github.com/getredash/redash/pull/2503

**Comment by rafrombrc** [comment]
The upstream PR seems to have stalled... next steps are to make a bump comment there to see if we can get some movement happening again.

**Comment by jezdez** [comment]
This was also filed as [bug 1463855](https://bugzilla.mozilla.org/show_bug.cgi?id=1463855).

**Comment by jezdez** [comment]
Correction, apparently https://github.com/getredash/redash/pull/2503 is only something that would concern us if we were to have more than 1000 database, which we don't.

All we need is https://github.com/getredash/redash/pull/2449 which upgrades boto and [adds a bunch of pagination support for various boto services](https://github.com/boto/botocore/commit/ae421f9077a66435b719ea723cdda59316eddf4b), apparently fixing this problem.

So in short: the next deploy of our current master will fix this.

**Comment by jezdez** [comment]
Thanks to @jasonthomas for the help figuring this out!
